runtime/j9vm/jvm.c

```diff
@@ -526,15 +526,31 @@ static const J9SignalMapping signalMap[] = {
 	{NULL, J9_SIG_ERR}
 };
 
-static void
-captureCommandLine(void)
-{
 #if defined(WIN32)
-#define ENV_VAR_NAME L"OPENJ9_JAVA_COMMAND_LINE"
+static BOOLEAN
```

Comment:
The two functions should have function comments.

Reply:
Added in 964388146557c42ac89287b600a58e021a679957.